src/System.Management.Automation/DscSupport/CimDSCParser.cs

```diff
@@ -2820,17 +2820,17 @@ private static string MapAttributesToMof(string[] enumNames, IEnumerable<object>
                 {
                     if (dscProperty.Key)
                     {
-                        sb.AppendFormat(CultureInfo.InvariantCulture, "{0}key", needComma ? ", " : "");
+                        sb.AppendFormat(CultureInfo.InvariantCulture, "{0}key", needComma ? ", " : string.Empty);
                         needComma = true;
                     }
                     if (dscProperty.Mandatory)
                     {
-                        sb.AppendFormat(CultureInfo.InvariantCulture, "{0}required", needComma ? ", " : "");
+                        sb.AppendFormat(CultureInfo.InvariantCulture, "{0}required", needComma ? ", " : string.Empty);
```

Comment:
Could migrate all of the `", "` strings into a constant.

Reply:
Constant or static?

Reply:
Is there any reason it shouldn't be constant?

Reply:
We already have a lot of static variables for such strings.
I found 4487 hits in 581 file for ", ". So static may be useful.

Reply:
As long as we don't need to change the string, I prefer constant. But that should be in a separate PR.